docs/repository_overview.rst

```diff
@@ -48,3 +49,7 @@ usmqe-centos-ci
 Upstream: https://github.com/usmqe/usmqe-centos-ci
 
 Machine deployment for Tendrl project in CentOS CI.
+
+.. TODO: fix the link
+.. _`main.yaml`: https://github.com/usmqe/usmqe-tests
+.. _`defaults.yaml`: https://github.com/usmqe/usmqe-tests
```

Comment:
You don't have to link the config files like that here, but if you do, edit the urls already so that it will work when merged.